tailwind.config.js

```diff
@@ -6,6 +6,23 @@ module.exports = {
         autodesktop: "repeat(auto-fit, minmax(410px, 1fr))",
         automobile: "repeat(auto-fit, minmax(300px, 1fr))"
       },
+      container: {
```

Comment:
_Adding this early while still draft_

In this file: grid, container and screens is not correct - we should not use grid as we decided on flex cross platform, container classes rewrite the existing default behaviour without improving it (having the padding removed could be perceived as limiting), the screens section is overwriting 2 breakpoints without any documentation.

The container class is supposed to come with responsive padding, setting it to 15px is making decisions that lead to technical debt. If this PR is implementing responsive containers, all those 3 top level keys should be removed, rather than patch-worked on previous rogue decisions.